app/services/dwp/benefit_check_service.rb

```diff
@@ -1,10 +1,71 @@
 module DWP
   class BenefitCheckService
+    BENEFIT_CHECKER_NAMESPACE = 'https://lsc.gov.uk/benefitchecker/service/1.0/API_1.0_Check'.freeze
+    USE_MOCK = ActiveModel::Type::Boolean.new.cast(ENV.fetch('BC_USE_DEV_MOCK', nil))
```

Comment:
We probably don't want to use constants for ENV variables. Main reason is they are more difficult to mock in tests.
Better to use just a method that encapsulates the ENV.fetch (i.e. `#use_mock?`)